engine/src/test/java/com/redhat/cloud/notifications/events/LifecycleITest.java

```diff
@@ -461,7 +463,7 @@ void addEventTypeBehavior(UUID eventTypeId, UUID behaviorGroupId) {
 
     @Transactional
     void clearEventTypeBehaviors(EventType eventType) {
-        entityManager.createQuery("DELETE EventTypeBehavior WHERE eventType = :eventType")
+        entityManager.createQuery("DELETE FROM EventTypeBehavior WHERE eventType = :eventType")
```

Comment:
Was this wrong before? Or is just better (style-wise) to use the `FROM` ?

Reply:
I think Hibernate automatically added the missing `FROM` but IntelliJ complained about this and I wanted him to be happy 😄